content/kubermatic/master/_index.en.md

```diff
@@ -9,30 +9,73 @@ date = 2019-04-27T16:06:34+02:00
 
-## Want to start with KKP in a GitOps way?
-Check the [Start with KKP]({{< ref "./installation/startio/" >}}) for more details.
 
 ## Features
 
-{{% notice tip %}}
-For latest updates follow us on Twitter [@Kubermatic](https://twitter.com/Kubermatic)
-{{% /notice %}}
+#### Powerful & intuitive dashboard to visualize your Kubernetes deployment
+Manage your projects and clusters with the KKP dashboard. Scale your cluster by adding and removing nodes in just a few clicks. As an Admin, the dashboard also allows you to customize the theme and disable theming options for other users.
+
+#### Deploy, scale & update multiple Kubernetes clusters
+Kubernetes environments must be highly distributed to meet the performance demands of modern cloud native applications. Organizations can ensure consistent operations across all environments with effective cluster management. KKP empowers you to take advantage of all the advanced features that Kubernetes has to offer and increases the speed, flexibility and scalability of your cloud deployment workflow. 
+
+At Kubermatic, we have chosen to do multi-cluster management with Kubernetes Operators. Operators (a method of packaging, deploying and managing a Kubernetes application) allow KKP to automate creation as well as the full lifecycle management of clusters. With KKP you can create a cluster for each need, fine-tune it, reuse it and continue this process hassle-free. This results in:
+- Better failure resilience
+- Smaller individual clusters being more adaptable than one big cluster
+- Faster development thanks to less complex environments
+
+#### Kubernetes Autoscaler
+Autoscaling in Kubernetes refers to the ability to increase or decrease the number of nodes as the demand for service response changes. Without autoscaling, teams would manually first provision and then scale up or down resources every time conditions change. This means, either services fail at peak demand due to the unavailability of enough resources or you pay at peak capacity to ensure availability. 
+
+The Kubernetes Autoscaler in the KKP Cluster automatically scales up/down when one of the following conditions is satisfied:
+1. Some pods fail to run in the cluster due to insufficient resources. OR
+2. There are nodes in the cluster that have been underutilized for an extended period (10 minutes by default) and can place their Pods on other existing nodes.
+
+#### Manage all KKP users directly from a single panel
+The Admin panel allows KKP administrators to manage the global settings that impact all KKP users directly. Here as an administrator, you can do the following:
+
+- Customize the way custom links (example: Twitter, Github, Slack) are displayed in the Kubermatic dashboard
+- Control various cluster related settings that influence cluster creation, management and clean up after deletion
+- Select from a range of filters to find and control existing dynamic data centres or add new ones
```

Comment:
What do you think about replacing this with something along this (some additional rewording is needed):

> Manage where users can deploy their clusters via the Dynamic Datacenters feature